app/External/Stripe/ProcessWebhookJob.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\External\Stripe;
+
+use App\StorableEvents\Stripe\IssuingAuthorization;
+use Spatie\WebhookClient\Models\WebhookCall;
+
+class ProcessWebhookJob extends \Spatie\WebhookClient\Jobs\ProcessWebhookJob
+{
+    public function __construct(WebhookCall $webhookCall)
+    {
+        parent::__construct($webhookCall);
+        $this->onQueue('webhooks');
+    }
+
+    public function handle()
+    {
+        if (is_null($this->webhookCall->topic)) {
+            // Probably an error
+            Log::error('Webhook call '.$this->webhookCall->id.' had no topic');
+
+            return;
+        }
+
+        $payload = $this->webhookCall->payload;
+
+        switch ($this->webhookCall->topic) {
+            case 'issuing_authorization.created':
+                event(new IssuingAuthorization($payload));
+//            Not Sure what to do to update an authorization if its already been created
+//            Would we just add it to the db as another event and handle it somewhere else?
```

Comment:
More or less, yeah. Depending on what all we want to do with this, it might make sense to talk about an aggregate. For example, we do that for each denhac membership since we need to process events and then make decisions about what to do next.